emsdk.py

```diff
@@ -162,17 +162,6 @@ def exit_with_error(msg):
 KEEP_DOWNLOADS = bool(os.getenv('EMSDK_KEEP_DOWNLOADS'))
 
 
-def os_name():
-  if WINDOWS:
-    return 'win'
-  elif LINUX:
-    return 'linux'
-  elif MACOS:
-    return 'macos'
-  else:
-    raise Exception('unknown OS')
-
-
 def os_name_for_emscripten_releases():
```

Comment:
I suppose we could just rename this to `os_name()`

Reply:
But I guess we'd want to document that it's intended to match what's used by the emscripten-releases builder

Reply:
Good idea.. done